cost-analyzer/templates/cost-analyzer-network-costs-template.yaml

```diff
@@ -18,6 +18,7 @@ spec:
         app: {{ template "cost-analyzer.networkCostsName" . }}
     spec:
       hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
```

Comment:
@mbolt35 @AjayTripathy any reason to believe this would have impact outside of network costs pods?

Reply:
@dwbrown2 Good catch here. I don't think this will have an impact on anything currently, but it wasn't really supposed to be included. I think the default dnsPolicy is fine in this case.